src/backend/metal/src/command.rs

```diff
@@ -2210,11 +2258,38 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
             }
             _ => CommandSink::Deferred {
                 is_encoding: false,
+                is_inheriting: info.subpass.is_some(),
                 journal: inner.backup_journal.take().unwrap_or_default(),
             },
         };
         inner.sink = Some(sink);
-        self.state.reset_resources();
```

Comment:
Does dropping `self.state.reset_resources()` here impact the non-SCB case? Was it not necessary before?

Reply:
I dropped it because it's already done in `self.reset()` :)